### PR TITLE
Feat : Search functionality

### PR DESCRIPTION
**Pull request by anuj-kumary**
Implement Filter Functionality
  - Search With Label
  - Sort By Date
  
  Preview:- https://deploy-preview-7--eagle-notes.netlify.app/

### CODE REVIEW

**Review by PeeyushPrashant — APPROVED** [review]
looks good to merge

File: src/components/Main/Main.jsx [hunk 1]
```diff
@@ -9,6 +9,7 @@ import './Main.css';
 
 export const Main = () => {
   const [expand, setExpand] = useState(false);
+  //const [pinned, setPinned] = useState(false);
   const { token } = useAuth();
```
Comment (samyakshah3008) [inline src/components/Main/Main.jsx]: Can remove this commented code

File: src/components/Navbar/Navbar.css [hunk 2]
```diff
@@ -25,4 +25,29 @@ body {
 
 .navbar__social-link :hover {
     color: #fff;
+}
```
Comment (samyakshah3008) [inline src/components/Navbar/Navbar.css]: Can use css variables

File: src/components/Navbar/Navbar.css [hunk 3]
```diff
@@ -25,4 +25,29 @@ body {
...
+    border-width: 0;
+    border: 1px solid;
+    padding-left: .5rem;
+}
+
+.filter__icon {
+    position: absolute;
+    top: .8rem;
+    right: .4rem;
+    color: black;
```
Comment (samyakshah3008) [inline src/components/Navbar/Navbar.css]: Can use color variables and shorthand property for top right

**Review by samyakshah3008 — APPROVED** [review]
Given some suggestions, can refactor later on 
Rest is all good for approval ✅

File: src/components/Navbar/Navbar.css (hunk 3 above)
Comment (anuj-kumary) [reply src/components/Navbar/Navbar.css]: Done

File: src/components/Navbar/Navbar.css (hunk 2 above)
Comment (anuj-kumary) [reply src/components/Navbar/Navbar.css]: Done

File: src/components/Main/Main.jsx (hunk 1 above)
Comment (anuj-kumary) [reply src/components/Main/Main.jsx]: Done

**Pull request merged**